secret-rotator/svckey/svckey.go

```diff
@@ -47,18 +47,24 @@ func (svc ServiceAccountKeySpec) Labels() map[string]string {
 }
 
 type Provisioner struct {
-	Service *iam.Service
+	// if EnableDeletion is set to true, the provisioner deletes the old svc key of 'version'
+	// when Deactivate('labels', 'version') is called.
+	EnableDeletion bool
```

Comment:
is there a reason for this to be public yet also a constructor parameter?

Reply:
I think I'll make it private, since it wouldn't be accessed outside of this struct.